erpnext/buying/doctype/purchase_order/test_purchase_order.py

```diff
@@ -544,7 +539,34 @@ def test_make_purchase_invoice_with_terms(self):
 		self.assertEqual(
 			getdate(pi.payment_schedule[1].due_date), add_days(getdate(po.transaction_date), 30)
 		)
-		automatically_fetch_payment_terms(enable=0)
+
+	def test_make_purchase_invoice_with_individual_terms(self):
```

Comment:
This test expands on that, checking if Payment Terms are properly preserved even if edited in the PO.